openj9Settings.mk

```diff
@@ -126,10 +126,21 @@ else
 	ADD_JVM_LIB_DIR_TO_LIBPATH:=export $(TEST_LIB_PATH);
 endif
 
+#######################################
+# In JDK18+, java.security.manager == null behaves as -Djava.security.manager=disallow.
+# In JDK17-, java.security.manager == null behaves as -Djava.security.manager=allow.
+# For OpenJ9 tests to work as expected, -Djava.security.manager=allow behaviour is
+# needed in JDK18+.
+#######################################
+ifeq (18,$(word 1,$(sort 18 $(JDK_VERSION))))
+	EXTRA_OPTIONS += -Djava.security.manager=allow
+endif
```

Comment:
The above code does not seem to work as expected:

1. it sets `-Djava.security.manager=allow` for JDK8 
2. `-Djava.security.manager=allow` is added multiple times. See JVM_OPTIONS.

JDK8: https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/564/console
```
11:28:52  ===============================================
11:28:52  Running test floatSanityTests_2 ...
11:28:52  ===============================================
11:28:52  floatSanityTests_2 Start Time: Tue Jan 25 11:24:08 2022 Epoch Time (ms): 1643127848273
11:28:52  "/home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image/bin/java" -Xshareclasses:destroyAll; "/home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image/bin/java" -Xshareclasses:groupAccess,destroyAll; echo "cache cleanup done";
11:28:52  JVMSHRC005I No shared class caches available
11:28:52  JVMSHRC005I No shared class caches available
11:28:52  cache cleanup done
11:28:52  variation: Mode150
11:28:52  JVM_OPTIONS:  -XX:+UseCompressedOops -Xint -Djava.security.manager=allow -Djava.security.manager=allow -Djava.security.manager=allow -Djava.security.manager=allow
11:28:52  { itercnt=1; \
11:28:52  mkdir -p "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/output_16431278457721/floatSanityTests_2"; \
11:28:52  cd "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/output_16431278457721/floatSanityTests_2"; \
11:28:52  "/home/jenkins/workspace/Grinder/openjdkbinary/j2sdk-image/bin/java"  -XX:+UseCompressedOops -Xint -Djava.security.manager=allow -Djava.security.manager=allow -Djava.security.manager=allow -Djava.security.manager=allow \
11:28:52  -cp "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../../jvmtest/TestConfig/resources:/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/lib/testng.jar:/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/lib/jcommander.jar:/home/jenkins/workspace/Grinder/aqa-tests/TKG/../../jvmtest/functional/Java8andUp/GeneralTest.jar" \
11:28:52  org.testng.TestNG -d "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/output_16431278457721/floatSanityTests_2" "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../../jvmtest/functional/Java8andUp/testng.xml" \
11:28:52  -testnames floatSanityTests \
11:28:52  -groups level.sanity \
11:28:52  -excludegroups d.*.linux_x86-64,d.*.arch.x86,d.*.os.linux,d.*.bits.64,d.*.generic-all; \
11:28:52  if [ $? -eq 0 ] ; then echo ""; echo "floatSanityTests_2""_PASSED"; echo ""; cd /home/jenkins/workspace/Grinder/aqa-tests/TKG/..; rm -f -r "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/output_16431278457721/floatSanityTests_2"; else echo ""; echo "floatSanityTests_2""_FAILED"; echo ""; fi; } 2>&1 | tee -a "/home/jenkins/workspace/Grinder/aqa-tests/TKG/../TKG/output_16431278457721/TestTargetResult";
11:28:53  Exception in thread "main" java/lang/InternalError: Could not create SecurityManager: allow
11:28:53  	at sun/misc/Launcher.<init> (Launcher.java:106)
11:28:53  	at sun/misc/Launcher.<clinit> (Launcher.java:54)
11:28:53  	at java/lang/ClassLoader.initializeClassLoaders (ClassLoader.java:196)
11:28:53  	at java/lang/Thread.initialize (Thread.java:405)
11:28:53  	at java/lang/Thread.<init> (Thread.java:154)
11:28:53  
11:28:53  floatSanityTests_2_FAILED
```

JDK18: https://openj9-jenkins.osuosl.org/view/Test/job/Grinder/568/console
```
11:43:54  JVM_OPTIONS:  -Xint -Djava.security.manager=allow -Djava.security.manager=allow -Djava.security.manager=allow -Djava.security.manager=allow
```